docs/docs/guides/tools/saved-views.md

```
@@ -0,0 +1,173 @@
+3. Click the view that you want to access. The saved view displays in the **Traces** or **Evals** tab. 
+
+### Rename a saved view
+
+1. Follow the steps described in [Access a saved view](#access-a-saved-view).
+2. In the upper lefthand corner of the **Traces** or **Evals** tab, click the view name.
+3. Enter a new name for the view.
+4. To save the new view name, press **Enter**.
+
+### Edit a saved view
+
+1. Follow the steps described in [Access a saved view](#access-a-saved-view).
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)